docs/release_notes.adoc

```diff
@@ -8,12 +8,12 @@ include::include.adoc[]
 === Groovy-3.0 Support
 
 The main feature of this milestone is support for Groovy 3.
-To use Spock in your Groovy 3 project just select the `spock-\*-2.0-M2` artifact(s) ending with `-groovy-3.0`.
+To use Spock in your Groovy 3 project just select the `spock-*-2.0-M2` artifact(s) ending with `-groovy-3.0`.
 
-NOTE: As Groovy 3 is not backward compatible with Groovy 2, there is a layer of abstraction in Spock to allow to build (and use) the project with both Groovy 2 and 3.
-      As a result, an extra artifact `spock-groovy2-compat` is (automatically) used in projects with Groovy 2.
-      It is *very important* to *do not mix* the `spock-\*-2.x-groovy-2.5` artifacts with the `groovy-*-3.x` artifacts on a classpath.
-      This may result in weird runtime errors.
+NOTE: As Groovy 3 is not backward compatible with Groovy 2, there is a layer of abstraction in Spock to allow
+to build (and use) the project with both Groovy 2 and 3. As a result, an extra artifact `spock-groovy2-compat` is
+(automatically) used in projects with Groovy 2. It is *very important* to *do not mix* the `spock-\*-2.x-groovy-2.5`
+artifacts with the `groovy-*-3.x` artifacts on a classpath. This may result in weird runtime errors.
```

Comment:
btw the `groovy-*-3.x` is wrong too, it should be `spock-*-groovy-3.0`

Reply:
Not, it is ok. `spock-groovy2-compat` contains the same 2 classes as `groovy-core-3.x` itself (which are not present in Groovy 2) to make the execution not fail at runtime.

Of course the spock-1.3 and spock-2.0 artifacts should not be mixed as well, but I assume it's obvious for the users.

Reply:
ah ok, I think the `*` is more confusing here, maybe just say `Groovy 3.x` instead, otherwise it looks too similar IMHO